pkg/util/util.go

```diff
@@ -84,6 +84,17 @@ func FileExists(filename string) (bool, error) {
 	return true, nil
 }
 
+// Verify block device is not in use by calling openFile with O_EXCL flag.
+// Returns nil if device not in use, but a non-nil return value means
+// that either the device is in use or another error occurred during OpenFile.
+func CheckDeviceInUseBySystem(pathname string) error {
```

Comment:
I originally had it returning two values, but found that equally if not more confusing. As far as I could find there is no clean function to determine if an error is ebusy or something else (e.g. the file does not exist), so when error == nil you can be sure that this returns false, but error != nil could mean the device is in use (true) or any number of other errors (false?). So I thought that just returning an error which could then be evaluated by the caller could be ported to other possible use cases outside of this one. Maybe it would be more clear if we renamed the function to something like TryExclusiveOpen?